### PR TITLE
Feature/password generator

### PR DESCRIPTION
**Pull request by iamnishant14**
Added plugin to generate different types of passwords of specified length.

### CODE REVIEW

File: honeybot/plugins/password_generator.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+[Commands]
+>>> .passgen <<length>>
+returns alphabetic password of specified length
+
+>>> .passgensecure <<length>>
+returns secure alphanumeric password of specified length
+
+>>> .passgenalphanum <<length>>
+returns alphanumeric password of specified length
+
```
Comment (Abdur-rahmaanJ) [inline honeybot/plugins/password_generator.py]: excellent, had in mind to gen different pswd but forgot to put in descrip. maybe a nice addition might be: secure easy to remember password. like i-lov3@swimming4lif3

File: honeybot/plugins/password_generator.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+>>> .passgenspecialchar <<length>>
+returns universally accepted alphanumeric password with special characters of specified length where length >= 4
+"""
+
+import string, random, secrets
+
```
Comment (Abdur-rahmaanJ) [inline honeybot/plugins/password_generator.py]: +1 use of string

File: honeybot/plugins/password_generator.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                        methods['send'](info['address'], self.__passgensecure(length))
+                    if msgs[0] == '.passgenalphanum':
+                        length = msgs[1]
+                        methods['send'](info['address'], self.__passgenalphanum(length))
+                    if msgs[0] == '.passgenspecialchar':
+                        length = msgs[1]
+                        if length >= 4:
+                            methods['send'](info['address'], self.__passgenspecialchar(length))
+                        else:
+                            raise Exception('Length od password should be greater than 4.')
```
Comment (Abdur-rahmaanJ) [inline honeybot/plugins/password_generator.py]: woops typo, length 'of'

Comment (iamnishant14) [reply honeybot/plugins/password_generator.py]: would be fixing in next PR

File: honeybot/plugins/password_generator.py (hunk 1 above)
Comment (iamnishant14) [reply honeybot/plugins/password_generator.py]: secure easy to remember password?

Comment (Abdur-rahmaanJ) [reply honeybot/plugins/password_generator.py]: like it generates words but modifies to something similar to i-lov3@swimming4lif3 such that it contains alphanum + symbols. find replacement for i etc in symbols like we replaced e by 3

Comment (Abdur-rahmaanJ) [reply honeybot/plugins/password_generator.py]: opening an issue for that

**Pull request merged**